SLA liquidity (#1685)
vegaprotocol/specs: protocol/0014-ORDT-order_types.md

1. **Reduce-Only (True/False):** Only valid for Non-Persistent orders. Cannot be True at the same time as Post-Only. If set, order will only be executed if the outcome of the trade moves the trader's position closer to 0. In addition, a Reduce-Only order will not move a position to the opposite side to the trader's current position (e.g. if short, a Reduce-Only order cannot make the trader long as a result). If submitted as IOC, where the full volume would switch sides, only the amount required to move the position to 0 will be executed.

### 'Glassberg' transparent iceberg orders

> On centralised exchanges, icebergs are a type of order that enables a trader to make an order with a relatively small visible "display quantiy" and a larger hidden total size.

Author: ```suggestion
On centralised exchanges, icebergs are a type of order that enables a trader to make an order with a relatively small visible "display quantity" and a larger hidden total size.
```

> Like an iceberg, most of the order is not visible to other traders.

Author: ```suggestion
Like an iceberg, most of the order is not active on the order book.
```

> After the full size of the visible portion of the order has traded away, the order is "refreshed" and reappears with its maximum display quantity.

Author: ```suggestion
After the full active portion (specified amount, see below) of the order has traded away, the order is "refreshed" and re-activates with its maximum display quantity.
```

This is repeated until the order is cancelled or expires, or its full volume trades away.

> Some platforms also allow the display quantity to be randomised on each refresh to further frustrate traders trying to identify the existence of the iceberg ordeer).

Author: ```suggestion
Some platforms also allow the display quantity to be randomised on each refresh to further frustrate traders trying to identify the existence of the iceberg ordeer).
```
